boltdesignsystem/bolt
# DS-118: remove unused height utils and fix docs

## Jira

https://pegadigitalit.atlassian.net/browse/DS-118

## Summary

Removed xxsmall to xxlarge height utility classes.

## Details

1. Removed the options in CSS.
2. Updated docs to not used the removed classes.
3. Updated small layout quirks on code snippet and tooltip doc page.

## How to test

Run the branch locally and make sure these classes no longer work:

1. `.u-bolt-height-xxsmall`
1. `.u-bolt-height-xsmall`
1. `.u-bolt-height-small`
1. `.u-bolt-height-medium`
1. `.u-bolt-height-large`
1. `.u-bolt-height-xlarge`
1. `.u-bolt-height-xxlarge`

And their responsive options such as: `.u-bolt-height-xxsmall@[breakpoint]`.

## Release notes

The following utility classes are no longer supported. 

1. `.u-bolt-height-xxsmall`
1. `.u-bolt-height-xsmall`
1. `.u-bolt-height-small`
1. `.u-bolt-height-medium`
1. `.u-bolt-height-large`
1. `.u-bolt-height-xlarge`
1. `.u-bolt-height-xxlarge`

And their responsive options such as: `.u-bolt-height-xxsmall@[breakpoint]`.
